Modules/_curses_panel.c

```diff
@@ -622,57 +644,75 @@ static PyMethodDef PyCurses_methods[] = {
 };
 
 /* Initialization function for the module */
-
-
-static struct PyModuleDef _curses_panelmodule = {
-        PyModuleDef_HEAD_INIT,
-        "_curses_panel",
-        NULL,
-        sizeof(_curses_panelstate),
-        PyCurses_methods,
-        NULL,
-        _curses_panel_traverse,
-        _curses_panel_clear,
-        _curses_panel_free
-};
-
-PyMODINIT_FUNC
-PyInit__curses_panel(void)
+static int
+_curses_exec(PyObject *mod)
```

Comment:
Since Python also has a _curses module, I would prefer to rename the function to _curses_panel_exec() to avoid confusion (to be able to put a breakpoint on it in a debugger).